ncm-ssh/src/main/pan/components/ssh/schema.pan

```diff
@@ -5,7 +5,7 @@
 
 declaration template components/ssh/schema;
 
-include 'quattor/types/component';
+include 'quattor/schema';
```

Comment:
what do you need that is not in the component template? we should try to keep this minimal.